docsite/rst/developing_plugins.rst

```diff
@@ -88,7 +88,7 @@ and is guaranteed to entertain and/or annoy coworkers.
 Configuring
 +++++++++++
 
-To activate a callback drop it in a callback directory as configured in :ref:`ansible.cfg <callback_plugins>`.
+To activate a callback drop it in a callback directory as configured in :ref:`ansible.cfg <callback_plugins>`. As of version 2.0 of ansible certain plugins will require you to add the plugin to a whitelist before it will execute. You can adjust the whitelist in :ref:`ansible.cfg <callback_whitelist>`. Plugin load order is alphanumeric in nature. If you have a plugin you want to run first consider naming it `1_first.py`, or if you have a plugin you want to run last consider naming it `z_last.py`. 
```

Comment:
whitelist is only for callbacks, also this is documented in the setting, no need to add here, also its not for ALL, only ones that specifically ask for it (shipped with ansible mainly).